tests/Doctrine/Tests/ORM/Functional/Locking/LockTest.php

```diff
@@ -35,13 +37,16 @@ public function testLockVersionedEntity() {
         $this->_em->flush();
 
         $this->_em->lock($article, LockMode::OPTIMISTIC, $article->version);
+
+        $this->addToAssertionCount(1);
```

Comment:
I didn't find a way to do it since we just compare the version value thrown an exception when it's different than the expected